modules/core/test/v2/unit/coins/abstractEthCoin.ts

```diff
@@ -57,6 +57,10 @@ describe('ETH-like coins', () => {
         gasPrice = '10000',
         gasLimit = '20000',
       }) {
+        // TODO : Remove this when account lib has changed celo to cgld
```

Comment:
Something isn't quite working right, the monorepo is designed to allow cross-package changes like this. Something I'll have to look into later.